src/main/kotlin/uk/gov/gdx/datashare/config/OpenApiConfiguration.kt

```diff
@@ -71,12 +75,21 @@ class OpenApiConfiguration(
             .openIdConnectUrl("$issuerUri/.well-known/openid-configuration"),
         ),
     )
-    .addSecurityItem(SecurityRequirement().addList("bearer-jwt", listOf("read", "write")))
+    .addSecurityItem(SecurityRequirement().addList("bearer-jwt"))
     .addSecurityItem(SecurityRequirement().addList("cognito"))
 
   @Bean
   fun openAPICustomiser(): OpenApiCustomizer = OpenApiCustomizer {
+    it.paths.forEach { (_, path: PathItem) ->
+      path.readOperations().forEach { operation ->
+        operation.responses.default = createErrorApiResponse("Unexpected error")
+        operation.responses.addApiResponse("401", createErrorApiResponse("Unauthorized"))
+        operation.responses.addApiResponse("403", createErrorApiResponse("Forbidden"))
+        operation.responses.addApiResponse("429", createErrorApiResponse("Too many requests"))
```

Comment:
Could we add something in here for the 406 and 415s?

Reply:
Not sure 415 applies in all cases but 406 yes